code/online_challenges/src/hackerrank/Electronics_Shop/README.md

```diff
@@ -0,0 +1,9 @@
+<b> Problem Link</b>
```

Comment:
Please use proper markdown formatting. Look at this [README](https://github.com/OpenGenus/cosmos/blob/master/code/online_challenges/src/hackerrank/encryption/README.md) for example.